main.py

```
@@ -0,0 +1,185 @@
+        
+        # Ollama接続テスト
+        print("🦙 Ollama接続テスト...")
+        try:
+            import requests
+            response = requests.get("http://localhost:11434/api/tags", timeout=5)
+            if response.status_code == 200:
+                models = response.json().get("models", [])
+                print(f"✅ Ollama接続成功 (モデル数: {len(models)})")
+            else:
+                print("⚠️ Ollama接続失敗")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)